samples/data-sealing/enc1/Makefile

```diff
@@ -28,8 +28,8 @@ all:
 build:
 	@ echo "Compilers used: $(CC), $(CXX)"
 	oeedger8r ../datasealing.edl --trusted  --trusted-dir ../common
-	$(CXX) -g -c $(CXXFLAGS) $(INCLUDES) -std=c++11 ecalls.cpp ../common/dispatcher.cpp ../common/keys.cpp
-	$(CC) -g -c $(CFLAGS) $(CINCLUDES) ../common/datasealing_t.c
+	$(CXX) -g -c $(CXXFLAGS) $(INCLUDES) -DOE_API_VERSION=2 -DOE_BUILD_ENCLAVE -std=c++11 ecalls.cpp ../common/dispatcher.cpp ../common/keys.cpp
```

Comment:
I am somewhat surprised that `DOE_BUILD_ENCLAVE` is needed... that doesn't come from `CXXFLAGS`?

Reply:
My bad on this one, I copied over the defines that I saw when I added cmake files from the source tree.
Removing the define in the next rebase.